README.md

```diff
@@ -93,6 +93,12 @@ as ``ovirt_engine_setup_answer_file_path``.
 | ovirt_engine_setup_provider_ovn_username | admin@internal     | Username for OVN. |
 | ovirt_engine_setup_provider_ovn_password | UNDEF              | Password for OVN. |
 
+* Apache releated options:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_apache_config_root_redirection | true               | If `False` the apache will not be set as default page of the web server.   |
+| ovirt_engine_setup_apache_config_ssl | true     | If `False` ssl will need to be configured manually. |
```

Comment:
true -> True

Reply:
Description: If 'False', engine-setup will not configure Apache SSL settings, so administrators will need to configure them manually